.bootstraprc

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
I think this file should be bootstrap version specific.

maybe .bootstraprc-4.0

And we should allow a the specification of a custom one in case there's a new beta version of bootstrap or some other extensions.